W 12658699 processautomation runconfig consoleuser se (#154)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

* If you want to use the values of xref:processautomation-prepare-deployment-variable.adoc[global variables] or xref:processautomation-prepare-deployment-credential.adoc[credentials] as initial values for activity parameters, an RPA administrator must create them in the *Global Variables* or *Credential Pool* view first.
* A project in test or production phase must exist in the *Project Management* view of the *Process Automation* module.
* Obtain the needed permissions as a member of the process team in the project data sheet in the *Project Management* view of the *Process Automation* module.

> * To run a process in an existing console or RDP session, you must deploy it to a bot that is configured for console sessions and the specified user must be logged in in this session when you deploy the run configuration. If the user is logged in more than once, the bot starts the process in the first session it finds the user logged in to. The bot searches for the user in the order real console session, RDP session 1 to n. Do not close or minimize the session while the bot executes the process.

Author: Because the syntax is ambiguous, I changed the last sentence to:
The bot searches for the user in this order:
1. Physical console session
2. RDP session 1
3. RDP session ...
4. RDP session n
Do not close or minimize the session while the bot executes the process.

> * To run a process in a hidden session, you must deploy it to a bot configured for secure sessions. Before deploying a run configuration on a bot that has only one secure session licensed, do not disconnect the session, but sign out the specified user. Do not sign in to a computer where a bot with one licensed secure session is running a process.

Reviewer: I suggest reversing the sign-out text and the "do not disconnect the session" text. I always think it's better to put the positive text first:

"...sign out the specified user but do not disconnect the session."

Author: I left this as was because disconnecting by clicking *Disconnect* does not sign out the user. This is a bad habit of our users that leads to difficult to trace errors in running a process.

+
The variables are typed. You can only enter or link values of the same type. The values entered here are only start values and they can be overwritten by other values during the course of the process run.
* *User Task*
+
User tasks are elements of a process model, which cannot be processed fully automatically by a bot, but which require the support of a human. Specify which users are to process the user tasks by clicking on the *User Assignment or  User Group Assignment* icon. The number of assigned users and user groups is displayed in the table. You can specify both users and user groups at the same time. The user task is then displayed to each specified user until a user reserves it for processing.
+
Select the checkbox *Send email notification to all related users* when the task is ready for processing if you want all specified users to receive an email as soon as they are supposed to process a user task. This email contains the name of the user task, the process, and the project manager, as well as a prompt to review the pending tasks in the *My RPA* module.
* *Execution*
+

> Decide whether the test run is to be carried out in an existing console or RDP session or in a secure session, and specify an appropriate user.

Reviewer: ```suggestion
Decide whether to carry out the test run in an existing console or RDP session or in a secure session, and specify an appropriate user.
```

+

> You can link the user with a global credential variable. To do this,  click on the *Link a credential from credential pool* icon.

Reviewer: ```suggestion
You can link the user with a global credential variable by clicking on the *Link a credential from credential pool* icon.
```

+
Use the syntax `domain\username` instead of `username@domain.com` for domain users and `username` for local users, regardless of whether you enter the username manually or link a global credential.
+

> Make sure the bot you want to execute the process is configured for the execution type, because you cannot save the test configuration otherwise.

Reviewer: ```suggestion
Ensure that the bot you want to execute the process is configured for the execution type because you cannot save the test configuration otherwise.
```

+
Select the checkbox *Send email notification to all related users* when the task is ready for processing if you want all specified users to receive an email as soon as they are supposed to process a user task. This email contains the name of the user task, the process, and the project manager, as well as a prompt to review the pending tasks in the *My RPA* module.
* *Execution*
+
Decide whether to carry out the test run in an existing console or RDP session or in a secure session, and specify an appropriate user.
+
You can link the user with a global credential variable by clicking on the *Link a credential from credential pool* icon.
+
Use the syntax `domain\username` instead of `username@domain.com` for domain users and `username` for local users, regardless of whether you enter the username manually or link a global credential.
+
Ensure that the bot that you want to execute the process is configured for the execution type, because you cannot save the test configuration otherwise.
+

> The bot can overtake th console or RDP session only if the specified user is logged in at runtime. If no user or another user is logged in at runtime, the process run results show the failure *No active session with the user '<user name>' could be found on the system*. 

Reviewer 2: The bot can overtake **_the_** console

[[production-configuration-link-globals]]
Alternatively, you can set the value of an activity parameter by linking it with a global variable. To link the parameter to a variable, click the *Link a global activity parameter* icon image:linkglobalvariable-icon.png[symbol,1.5%,1.5%] and select a variable. RPA Manager displays variables of the same type, assigned to the production phase, and only the credentials to which you are xref:processautomation-prepare-deployment-credential.adoc#form-create-credential-useraffiliation[affiliated]. If a credential affiliated to another user is already linked, RPA Manager displays a row of stars to hide the data. 
+

> The variables are typed. You can only enter or link values of the same type. The values entered here are only start values and they can be overwritten by other values during the course of the process run.

Reviewer: ```suggestion
The variables are typed. You can only enter or link values of the same type. The values entered here are start values, which can be overwritten by other values during the course of the process run.
```

* *User Task*:
+
User tasks are elements of a process model, which cannot be processed fully automatically by a bot, but which require the support of a human. Specify which users are to process the user tasks by clicking on the *User Assignment or  User Group Assignment* icon. The number of assigned users and user groups is displayed in the table. You can specify both users and user groups at the same time. The user task is then displayed to each specified user until a user reserves it for processing.
+
Select the checkbox *Send email notification to all related users* when the task is ready for processing if you want all specified users to receive an email as soon as they are supposed to process a user task. This email contains the name of the user task, the process, and the project manager, as well as a prompt to review the pending tasks in the *My RPA* module.
* *Execution*:
+

> Decide whether the productive run is to be carried out in an existing console or RDP session or in a secure session, and specify an appropriate user.

Reviewer: ```suggestion
Decide whether to carry out the productive run in an existing console or RDP session or in a secure session, and specify an appropriate user.
```

+
You can link the user with a global credential variable. To do this,  click on the *Link a credential from credential pool* icon.
+
Use the syntax `domain\username` instead of `username@domain.com` for domain users and `username` for local users, regardless of whether you enter the username manually or link a global credential.
+

> Make sure the bot you want to execute the process is configured for the execution type, because you cannot save the run configuration otherwise.

Reviewer: ```suggestion
Ensure that the bot you that want to execute the process is configured for the execution type because you cannot save the run configuration otherwise.
```

+

> The console or RDP session can only be overtaken by the bot if the specified user is logged in at run time. If no user or another user is logged in at run time, the process run results show the failure *No active session with the user '<user name>' could be found on the system*. 

Reviewer: ```suggestion
The console or RDP session can be overtaken by the bot only if the specified user is logged in at runtime. If no user or another user is logged in at runtime, the process run results show the failure *No active session with the user '<user name>' could be found on the system*. 
```

Set the value by entering text or by clicking on a checkbox, and complete your entries by pressing ENTER.
+
[[invokable-configuration-link-globals]]

> Alternatively, you can set the values of the activity parameters by linking them with a global variable. To do this, click on the *Link a global activity parameter* icon. Only those variables assigned to the phase are displayed for linking. The variables are typed. You can only enter or link values of the same type. The values entered here are only start values and they can be overwritten by other values during the course of the process run.

Reviewer: ```suggestion
Alternatively, you can set the values of the activity parameters by linking them with a global variable. To do this, click on the *Link a global activity parameter* icon. Only those variables assigned to the phase are displayed for linking. The variables are typed. You can only enter or link values of the same type. The values entered here are start values, which can be overwritten by other values during the course of the process run.
```

* *User Task*:
+
User tasks are elements of a process model, which cannot be processed fully automatically by a bot, but which require the support of a human. Specify which users are to process the user tasks by clicking on the *User Assignment or  User Group Assignment* icon. The number of assigned users and user groups is displayed in the table. You can specify both users and user groups at the same time. The user task is then displayed to each specified user until a user reserves it for processing.
+
Select the checkbox *Send email notification to all related users* when the task is ready for processing if you want all specified users to receive an email as soon as they are supposed to process a user task. This email contains the name of the user task, the process, and the project manager, as well as a prompt to review the pending tasks in the *My RPA* module.
* *Execution*:
+

> Decide whether the invokable run is to be carried out in an existing console or RDP session or in a secure session, and specify an appropriate user.

Reviewer: ```suggestion
Decide whether to carry out the invokable run in an existing console or RDP session or in a secure session, and specify an appropriate user.
```

+

> You can link the user with a global credential variable. To do this,  click on the *Link a credential from credential pool* icon.

Reviewer: ```suggestion
You can link the user with a global credential variable. To do this, click on the *Link a credential from credential pool* icon.
```

+
Use the syntax `domain\username` instead of `username@domain.com` for domain users and `username` for local users, regardless of whether you enter the username manually or link a global credential.
+

> Make sure the bot you want to execute the process is configured for the execution type, because you cannot save the run configuration otherwise.

Reviewer: ```suggestion
Ensure that the bot that you want to execute the process is configured for the execution type because you cannot save the run configuration otherwise.
```

+

> The console or RDP session can only be overtaken by the bot if the specified user is logged in at run time. If no user or another user is logged in at run time, the process run results show the failure *No active session with the user '<user name>' could be found on the system*. 

Reviewer: ```suggestion
The console or RDP session can be overtaken by the bot only if the specified user is logged in at runtime. If no user or another user is logged in at runtime, the process run results show the failure *No active session with the user '<user name>' could be found on the system*. 
```
